Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeContext.mm

```diff
@@ -151,6 +152,8 @@
 
     PlatformCALayerRemote& rootLayerRemote = downcast<PlatformCALayerRemote>(rootLayer);
     transaction.setRootLayerID(rootLayerRemote.layerID());
+    if (m_webPage.parentFrameIdentifier())
```

Comment:
OK my head is starting to hurt with "parent frame", and "main frame"

Reply:
This will be resolved by future frame tree model improvements.